assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -257,6 +257,11 @@ che.infra.kubernetes.server_strategy=multi-host
 # - 'gateway': Exposes servers using reverse-proxy gateway.
 che.infra.kubernetes.singlehost.workspace.exposure=native
 
+# Expose devfile public endpoints on subdomans using dedicated Ingress/Route with single-host server_strategy.
+# 'true' - expose on subdomains
+# 'false' - expose on subpaths
+che.infra.kubernetes.singlehost.workspace.expose_devfile_endpoints_on_subdomains=false
```

Comment:
I've renamed it so many times, until ended with this :D yes, your's is better. However, I think we might need to do `devfile_endpoint_exposure`, because we might need some other property for plugins... wdyt?

Reply:
Thinking more about it, I would go with:

```
che.infra.kubernetes.singlehost.workspace.devfile_endpoint_exposure=(multihost|singlehost)
```

just to keep the naming consistent.

Reply:
yes, I made it even better with options `multi-host`/`single-host` so we can reuse existing constants :boom: